btcjson/chainsvrcmds.go

```diff
@@ -333,6 +333,19 @@ func NewGetInfoCmd() *GetInfoCmd {
 	return &GetInfoCmd{}
 }
 
+// GetMempoolEntryCmd defines the getmempoolentry JSON-RPC command.
+type GetMempoolEntryCmd struct {
+	TxID string
```

Comment:
Ah right, I noticed it was on one of the others (`GetHeadersCmd`).  Given it's positional though, you're right it isn't needed.